package_tracking/settings.py

```diff
@@ -124,12 +124,17 @@
 USE_TZ = True
 
 
-# Static files (CSS, JavaScript, Images)
-# https://docs.djangoproject.com/en/2.2/howto/static-files/
-
 STATIC_URL = '/static/'
-# Redirect to home URL after login (Default redirects to /accounts/profile/)
 LOGIN_REDIRECT_URL = '/'
 STATICFILES_DIRS = (
     os.path.join(BASE_DIR, 'static'),
 )
+
+# DataFlair
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_USE_TLS = True
+EMAIL_PORT = 587
+EMAIL_HOST_USER = 'testingmonke@gmail.com'
+STAFF_EMAILS = ['danielmkhlv@gmail.com']

```

Comment:
Instead of static destination, use something configurable in models. Say get all the users who are staff and send to all of them

Reply:
Yeah makes sense, I think there is something built in for sending emails to all staff users (the built in staff)

Reply:
MAKE IT DYNAMIC!!!

Reply:
![ok](https://i.imgur.com/jCLnBbR.png)
Format: ![ok](https://i.imgur.com/jCLnBbR.png)